client/reader/search-stream/index.jsx

```diff
@@ -63,18 +66,26 @@ const SearchCardAdapter = React.createClass( {
 		this.props.handleClick( this.props.post, {} );
 	},
 
+	onRefreshCardClick( post ) {
+		recordTrackForPost( 'calypso_reader_searchcard_clicked', this.props.post );
+		this.props.handleClick( post, {} );
+	},
+
 	onCommentClick() {
 		this.props.handleClick( this.props.post, { comments: true } );
 	},
 
 	render() {
-		return <SearchCard
-			post={ this.props.post }
-			site={ this.state.site }
-			feed={ this.state.feed }
-			onClick={ this.onCardClick }
-			onCommentClick={ this.onCommentClick }
-			showPrimaryFollowButton={ this.props.showPrimaryFollowButtonOnCards } />;
+		const isRefreshedStream = config.isEnabled( 'reader/refresh/stream' );
+		const cardComponent = isRefreshedStream ? ReaderPostCard : SearchCard;
```

Comment:
I'm not sure here, but it feels like this should be Capitalized, as it'll be referring to a component. 🤔